### DISCUSSION TITLE
Formgator, my new validation library

### DISCUSSION THREAD

**Discussion by giscus** [GauBen/gautier.dev#7] [Comments]
# Formgator, my new validation library

Discover the motivation and the development process that led to formgator, a  validation library for forms.

https://gautier.dev/articles/formgator

**Comment by mtsknn** [comment]
As a Zod fan, Formgator looks great! The readme looks clear.

I also enjoyed your musings on functional APIs and dev tools. I would like that functional API design even more if it didn't use the `this` keyword as it's such a big footgun. 😄

**Reply by GauBen** [reply]
Thank you! Unfortunately, it's not Gleam, we cannot enjoy the `|>` operator and a functional-by-design language 😄
